inst/app.crunch.io/api/datasets/1.json

```diff
@@ -29,7 +29,8 @@
         "table": "https://app.crunch.io/api/datasets/1/table/",
         "preferences": "https://app.crunch.io/api/datasets/1/preferences/",
         "settings": "https://app.crunch.io/api/datasets/1/settings/",
-        "pk": "https://app.crunch.io/api/datasets/1/pk/"
+        "pk": "https://app.crunch.io/api/datasets/1/pk/",
+		"stream": "https://app.crunch.io/api/datasets/1/stream/"
```

Comment:
This was some strange atom bug. In atom it looked totally fine, but anywhere else it was too tabbed. Should be fixed now.